client/reader/following/main.jsx

```diff
@@ -8,10 +8,8 @@ import React from 'react';
  */
 import Stream from 'reader/stream';
 
-const FollowingStream = ( props ) => {
-	return (
-		<Stream { ...props } />
-	);
-};
+const FollowingStream = props => (
+	<Stream { ...props } />
+);
```

Comment:
What's the point of this statement?

If we just want a different name then we can use

```js
import FollowingStream from 'reader/stream';
```